Loop/src/loop.hxx

```diff
@@ -517,6 +521,110 @@ public:
     } // for rank
   }
 
+  // Alex Dima:
+  // Modified from loop_bnd
+  // Loop over all "radiative" boundary points.
```

Comment:
I think this function is useful not just for radiative boundaries but for all kinds of evolved boundary conditions. Let's use a more general name.

Reply:
Good idea, do you have any suggestions for a more general name?

Reply:
You could ask on the ET channel.

Reply:
On the CarpetX gitter chat, Liwei Ji suggested `loop_int_outermost` and `loop_int_bnd`. Since it is not only the outermost single layer of points that gets looped over, but rather the outermost layer of thickness `nghostzones`, I prefer `loop_int_bnd` slightly over `loop_int_outermost`. However, it could be unclear what exactly int_bnd refers to without using a more specific name. What do you think?